playbook/app/pb_kits/playbook/pb_table/table_row.html.erb

```diff
@@ -17,3 +36,18 @@
       <%= content.presence %>
     <% end %>
 <% end %>
+
+<script>
+  if (!window.toggleCollapsibleRowById) {
+    window.toggleCollapsibleRowById = (id) => {
+      document.dispatchEvent(new CustomEvent(id));
+      
+      const toggleId = id.slice(-1);
+      const toggleElements = document.querySelectorAll(`.collapsible_border_toggle${toggleId}`);
+      toggleElements.forEach(element => {
+        element.classList.toggle('no-border');
+        element.classList.toggle('border-active');
+      });
+    };
+  }
+</script>
```

Comment:
@nickamantia, we should be able to move this to Enhanced Element and remove the need to use "document", which is causing performance issues and possible unexpected behaviors in the future.